app/components/SignUp/index.jsx

```diff
@@ -101,6 +101,16 @@ export default class SignUp extends React.Component {
       });
     }
 
+    if (this.state.name.length > 25) {
+      return this.setState({
```

Comment:
Such kind of restrictions on the client without the same limits on the server side just don't make any sense. I could set a breakpoint here and circumvent this protection, filling database with junk.
All kinds of security should be thought server-side first, and only then be reflected in the client interface.